src/api.ts

```diff
@@ -44,6 +44,7 @@ export default class Chromeless<T extends any> implements Promise<T> {
         closeTab: true,
         ...options.cdp,
       },
+      chromeFlags: ['--headless'],
```

Comment:
We _could_ go the route of passing in a hash of options:

```
{ headless: true } => '--headless'
{ remoteDebuggingPort: 1234 } => '--remote-debugging-port=1234'
{ headless: false, disableGpu: true } => '--disable-gpu'
```

This way you can splat new params into place, and still have defaults like `headless`